templates/todo/api/csharp-cosmos-sql/TodoEndPointsExt.cs

```diff
@@ -0,0 +1,143 @@
+﻿using Microsoft.AspNetCore.Http.HttpResults;
+using Microsoft.AspNetCore.Mvc;
+using SimpleTodo.Api;
+
+namespace Todo.Api
+{
+    public static class TodoEndpointsExt
+    {
+        public static void MapTodoEndpoints(this WebApplication app)

```

Comment:
Recommend targeting `IEndpointRouteBuilder` with this and returning the builder object to support chaining other calls at the invocation site.